core-bundle/src/Resources/contao/elements/ContentDownload.php

```diff
@@ -66,7 +66,7 @@ public function generate()
 		{
 			if ($file == $objFile->path)
 			{
-				\Controller::sendFileToBrowser($file);
+				\Controller::sendFileToBrowser($file, $this->disposition);
```

Comment:
You will need a fallback here for already existing elements in which case `$this->disposition` will be `null`.

Reply:
Existing elements get attachment as the db value, so this won't be necessary, no? Also in Controller::sendFileToBrowser I built in a default value for the parameter (`$strDisposition = ResponseHeaderBag::DISPOSITION_ATTACHMENT`).

Reply:
The default would not be in use, if you pass `null` as the second parameter.

Reply:
Right, but as mentioned before, we have the default value in the sql dca key. Would be enough, no?